src/checkbox/Checkbox.ts

```diff
@@ -98,29 +82,7 @@ export class CheckboxBase<P extends CheckboxProperties = CheckboxProperties> ext
 	}
 
 	protected renderToggle(): DNode[] {
```

Comment:
`renderToggle` can be removed from here

Reply:
It's necessary so it can be called in `render` and overridden in Toggle (which keeps Toggle from having to duplicate all the render code).